napari_allencell_annotator/controller/annotator_controller.py

```diff
@@ -23,11 +46,107 @@ def __init__(self, viewer: napari.Viewer):
         # 4 annotations
         # path = str(Directories.get_assets_dir() / "sample.json")
         # 8 annotations
-        path = str(Directories.get_assets_dir() / "sample2.json")
-        data = json.load(open(path))
+        path: str = str(Directories.get_assets_dir() / "sample2.json")
+        self.annot_data: Dict[str, Dict[str, str]] = json.load(open(path))
         # open in add mode
         # self.view = AnnotatorView(napari.Viewer(), data)
         # open in view mode
-        self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.VIEW)
-        # open in annotate mode
-        # self.view = AnnotatorView(viewer, data, mode=AnnotatorViewMode.ANNOTATE)
+        self.view: AnnotatorView = AnnotatorView(
+            viewer, self, mode=AnnotatorViewMode.VIEW
+        )
+        self.view.render_annotations(self.annot_data)
+        self.view.show()
+        self.curr_img: Dict[str, str] = None
+        self.csv_name: str = None
+
+        self.annotation_dict: Dict[str, (List[str], List[str])] = {}
```

Comment:
What do the lists store in this dictionary?

Reply:
I changed the type of this, but didn't update . It should be Dict[str, List[str]]. Fixed now